src/views/ifsBrowser.js

```diff
@@ -636,12 +636,19 @@ class Object extends vscode.TreeItem {
    * @param {"shortcut"|"directory"|"streamfile"} type
    * @param {string} label
    * @param {string} path
+   * @param {number} size
```

Comment:
```suggestion
   * @param {number} [size]
```

Will indicate that it is optional in jsdoc.